src/lib/cache/subCache.cpp

```diff
@@ -1277,12 +1284,12 @@ void subNotificationErrorStatus
     if (errors == 0)
     {
       // count == 0 (inc is done in another part), lastFailure == -1, failureReason == -1
-      mongoSubCountersUpdate(tenant, subscriptionId, 0, now, -1, now, "", statusCode);
+      mongoSubCountersUpdate(tenant, subscriptionId, 0, 0, now, -1, now, "", statusCode);
```

Comment:
Question: why failsCounter is 0 here? It happens the same as with count (`count == 0 (inc is done in another part)`)?

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/0d976f5d55b682e27c153ecd5662ef408f351a1e